hedera-node/src/main/resources/FeeSchedule.json

```diff
@@ -221,38 +221,38 @@
           "hederaFunctionality": "TokenUpdate",
           "feeData": {
             "nodedata": {
-              "constant": 1094986245,
-              "bpt": 1750555,
-              "vpt": 4376387182,
-              "rbh": 1167,
-              "sbh": 88,
-              "gas": 11670,
-              "bpr": 1750555,
-              "sbpr": 43764,
+              "constant": 23923923,
```

Comment:
Just curious, Are these changes because it is updated by pulling data from a file in network ?

Reply:
Yep, the _FeeSchedule.json_ was originally (almost) random for the token operations; this PR standardizes the prices for token ops based on the "closest" crypto ops.